tests/assets/move-projects/car-wash-example/sources/CarWash.move

```diff
@@ -44,7 +44,7 @@ module DeveloperBob::CarWash {
         move_to(account, Balance { coins: 0 });
     }
 
-    /// Transfers `amount` of tokens from `from` to `to`.
+    /// Buys a washing coin for the car wash. Therfor, `COIN_PRICE` will be withdrawn from the user's account.
```

Comment:
```suggestion
    /// Buys a washing coin for the car wash. Therefore, `COIN_PRICE` will be withdrawn from the user's account.
```

Reply:
Done, but will add this in another commit, because I wanted one single commit for the tutorial part and the other for the included example code and its unit-test